modules/core/src/main/scala/fs2/kafka/KafkaConsumer.scala

```diff
@@ -134,21 +127,8 @@ object KafkaConsumer {
     stopConsumingDeferred: TryableDeferred[F, Unit]
   )(implicit F: Concurrent[F]): KafkaConsumer[F, K, V] =
     new KafkaConsumer[F, K, V] {
-      override val fiber: Fiber[F, Unit] = {
-        val actorFiber =
-          Fiber[F, Unit](actor.join.guaranteeCase {
-            case ExitCase.Completed => polls.cancel
-            case _                  => F.unit
-          }, actor.cancel)
-
-        val pollsFiber =
-          Fiber[F, Unit](polls.join.guaranteeCase {
-            case ExitCase.Completed => actor.cancel
-            case _                  => F.unit
-          }, polls.cancel)
```

Comment:
This logic no longer exists in the updated PR but all the tests still pass - does this indicate a failure mode that we don't have a test for and should add tests? (@vlovgr)

Reply:
Good spot, there wasn't much logic in the test around the `fiber`. This is all I could find: https://github.com/fd4s/fs2-kafka/blob/series/1.x/modules/core/src/test/scala/fs2/kafka/KafkaConsumerSpec.scala#L203-L218

Reply:
As for the above block, that has been deleted, it seems a bit odd to me. I would have thought that if the `join` was successful that you shouldn't `cancel` the other `Fiber`. The `actorFiber combine pollsFiber` would `join` the `Fiber`s one after the other, so the `cancel` shouldn't be required (as far as I can tell).